titus-api/src/main/java/com/netflix/titus/api/jobmanager/model/job/PlatformSidecar.java

```diff
@@ -0,0 +1,131 @@
+package com.netflix.titus.api.jobmanager.model.job;
+
+import java.io.IOException;
+import java.util.Map;
+import java.util.Objects;
+import javax.validation.Valid;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.google.protobuf.Struct;
+import com.google.protobuf.util.JsonFormat;
+
+public class PlatformSidecar {
+
+    @Valid
+    private final String name;
+
+    @Valid
+    private final String channel;
+
+    @Valid
+    @JsonSerialize(using = ProtoStructSerializer.class)
+    @JsonDeserialize(using = ProtoStructDeserializer.class)
+//    @JsonSerialize(using = MyMessageSerializer.class)
+//    @JsonDeserialize(using = MyMessageDeserializer.class, as = Struct.class, keyAs = String.class, contentAs = Value.class)
+//    @JsonSerialize(as = Struct.class, keyAs = String.class, contentAs = Value.class)
+//    @JsonDeserialize(as = Struct.class, keyAs = String.class, contentAs = Value.class)
```

Comment:
experiments/testing leftovers?

Reply:
Yea :(
Disregard for now because I'm going to redo how we serialize after that meeting.